tests/AssertTraitTest.php

```diff
@@ -13,6 +13,18 @@
 
 class AssertTraitTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * Showcase for the Wiki.
+     *
+     * @see https://github.com/estahn/phpunit-json-assertions/wiki/assertJsonMatchesSchema
+     */
+    public function testAssertJsonMatchesSchemaSimple()
+    {
+        $content = json_decode(file_get_contents(Utils::getJsonPath('assertJsonMatchesSchema_simple.json')));
+
+        AssertTraitImpl::assertJsonMatchesSchema(Utils::getSchemaPath('assertJsonMatchesSchema_simple.schema.json'), $content);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'EnricoStahn\JsonAssert\Tests\AssertTraitImpl' in method 'testAssertJsonMatchesSchemaSimple'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2281688910/issues/source?bid=3181605&fileBranchId=3188493#l25)